### DISCUSSION TITLE
Transaction to an address in string

### DISCUSSION THREAD

**Discussion by web3-sante** [rankanizer/polkadart#440] [Q&A] [answered]
Hello team,
dumb question here:

How to execute a simple transfer to an address in this format 
```dart
    final BOB = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
```
without also having to grab infos like  [block hash, genesis...](https://github.com/leonardocustodio/polkadart/blob/main/examples/bin/extrinsic_demo.dart) but just something in this [style](https://polkadot.js.org/docs/api/examples/promise/make-transfer).

What would be the best way or place to ask questions without having to open issues?

**Comment by leonardocustodio** [comment]
Hello @web3-sante, right now, that's not possible. We started this project last year and there is a lot to be done yet. We are first working on supporting everything Polkadot can do, after that when people can do everything (in a harder or easier way) then we will start to work in the helpers and things to reduce the boilerplates. Feel free to send PR's if you make something and would like to contribute. And for now, issues here are the best (and only place), so no worries.

**Comment by web3-sante** [comment]
> Hello @web3-sante, right now, that's not possible. We started this project last year and there is a lot to be done yet. We are first working on supporting everything Polkadot can do, after that when people can do everything (in a harder or easier way) then we will start to work in the helpers and things to reduce the boilerplates. Feel free to send PR's if you make something and would like to contribute. And for now, issues here are the best (and only place), so no worries.

Thanks @leonardocustodio for the reply.

May be I asked the question the wrong way.

Assuming we are on polkadot how one would do to make a transfer instead of  [transferAll](https://github.com/leonardocustodio/polkadart/blob/main/examples/bin/extrinsic_demo.dart#L45C1-L46C1) ,  how to encode/decode the BOB address: 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty;  so we can use it in `api.tx.balances.transfer`

Something like this:
```dart
final BOB = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
final bobAddr =  ..... // Encoded Decoded
final transfer = api.tx.balances.transfer(dest: bobAddr, value: BigInt.from(11111) );
```

> Feel free to send PR's if you make something and would like to contribute. And for now, issues here are the best (and only place), so no worries.

 it's noted asap something is done on my side.

**Comment by leonardocustodio** [comment, marked as answer]
Ah I see, I thought you wanted a `signAndSend` method that would get the wallet from keyring, sign and send, without having to construct the SigningPayload and Extrinsic data.

 For the above, it is pretty simple:
```dart
import 'package:polkadart_example/generated/polkadot/polkadot.dart';
import 'package:ss58/ss58.dart';
import 'package:polkadart_example/generated/polkadot/types/sp_runtime/multiaddress/multi_address.dart';

final bob = Address.decode('5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty');
final runtimeCall = api.tx.balances.transfer(dest: MultiAddress.values.id(bob.pubkey), value: BigInt.from(1000000000000));
```

**Comment by web3-sante** [comment]
Awesome @leonardocustodio Thanks 🙏 !

**Comment by uddyami** [comment]
I'm trying this 
```
import 'package:polkadart/polkadart.dart';
import 'package:polkadart_example/generated/polkadot/polkadot.dart';
import 'package:polkadart_keyring/polkadart_keyring.dart';
import 'package:ss58/ss58.dart';
import 'package:polkadart_example/generated/polkadot/types/sp_runtime/multiaddress/multi_address.dart';

Future<void> main(List<String> arguments) async {
  final bob = Address.decode('5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty');
  final polkadotProvider = Provider.fromUri(Uri.parse('wss://rpc.polkadot.io'));

  final api = Polkadot(polkadotProvider);

  final runtimeCall = api.tx.balances.transfer(dest: MultiAddress.values.id(bob.pubkey), value: BigInt.from(1000000000000));
}

```
and get the following error 
`The method 'transfer' isn't defined for the type 'Txs'.`

**Reply by leonardocustodio** [reply]
Have you run `polkadart_cli` so it generates the classes?
